src/test/java/com/ecore/roles/api/MembershipsApiTests.java

```diff
@@ -21,7 +21,7 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
-public class MembershipsApiTests {
+class MembershipsApiTests {
```

Comment:
Default class modifier is the recomended way stated by JUnit: "The JUnit 5 team believe in the principle "Less is more", meaning the less you have to type to achieve your goal, the better!"